code/game/objects/items/storage/backpack.dm

```diff
@@ -314,13 +314,14 @@
 		icon_state += "_0"
 
 /obj/item/storage/backpack/marine/corpsman/MouseDrop_T(obj/item/W, mob/living/user) //Dragging the defib/power cell onto the backpack will trigger its special functionality.
-	if(istype(W, /obj/item/defibrillator))
+	var/datum/component/defiber/comp = W.GetComponent(/datum/component/defiber)
+	if(comp)
```

Comment:
I know you told me this is bad but other then creating a switch case for every future diffible item im not sure how else im supposed to make this.

Reply:
it is bad, TG uses a /atom/movable/proc/get_cell() but tha'd probably be out of scope to implement

Reply:
yandere deving it since that would be the only other choice